vlm/src/main/scala/geotrellis/contrib/vlm/gdal/GDALUtils.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2019 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.contrib.vlm.gdal
+
+import geotrellis.raster._
+import geotrellis.raster.io.geotiff._
+import geotrellis.raster.resample._
+
+
+object GDALUtils {
+  def deriveResampleMethodString(method: ResampleMethod): String =
```

Comment:
The same story about splitting things into geotrellis-gdal and geotrellis-contrib. @echeipesh